src/go/configinfo/service_info.go

```diff
@@ -891,3 +922,16 @@ func calculateStreamIdleTimeout(operationDeadline time.Duration, opts options.Co
 	operationIdleTimeout := operationDeadline + time.Second
 	return util.MaxDuration(operationIdleTimeout, opts.StreamIdleTimeout)
 }
+
+func parseStatusCodes(statusCodes string) ([]uint32, error) {
+	codeList := strings.Split(statusCodes, ",")
+	var codes []uint32
+	for _, codeStr := range codeList {
+		if code, err := strconv.Atoi(codeStr); err != nil {
```

Comment:
should we verify status code is in legal range: "> 300 and < 600"?

Reply:
Done. Made valid http status code check [100, 600).